ts/features/itwallet/presentation/components/ItwPresentationDetailsFooter.tsx

```diff
@@ -148,6 +156,14 @@ const IPatenteListItemAction = () => {
       icon="externalLink"
       label={label}
       onPress={() => {
+        trackAuthenticationStart(
+          serviceId,
+          serviceName,
+          serviceOrganizationName,
+          serviceOrganizationFiscalCode,
+          label,
+          "message_detail"
+        );
```

Comment:
What do you think about creating a specific function for this tracking, removing the need to have const values in here?
Also, I think `message_detail` is not correct for this specific use case. Should be something different, like "credential_detail".  What do you think?